lib/client/signalflow/request_manager.js

```diff
@@ -390,11 +390,34 @@ function RequestManager(options) {
     //destroy all open SSE connections
   }
 
+  function registerLiveTail(callbacks, overrideRequestId ){
+    var requestId = overrideRequestId || getRequestIdAndIncrement();
+    knownComputations[requestId] = {
+      callback: callbacks,
+      onMessage: router.onMessage,
+      transport: transports.WEBSOCKET,
+      requestId: requestId,
+      lastDataBatchTime: router.getLatestBatchTimeStamp,
+    };
+    var jobObject = {};
```

Comment:
I'd avoid calling this `jobObject` to remove the confusion with signalflow jobs. Honestly just do:

```js
activeSocket.send(JSON.strigify({
  type: 'registerlivetail',
  channel: requestId,
  query: params.query,
  throttleOptions: params.throttleOptions,  // Assuming you'll be passing this too?
});